sm_cnn/README.md

```diff
@@ -1,80 +1,149 @@
 ## SM model
 
 #### References:
-1. Aliaksei _S_everyn and Alessandro _M_oschitti. 2015. Learning to Rank Short Text Pairs with Convolutional Deep Neural Networks. In Proceedings of the 38th International ACM SIGIR Conference on Research and Development in Information Retrieval (SIGIR '15). ACM, New York, NY, USA, 373-382. DOI: http://dx.doi.org/10.1145/2766462.2767738
+1. Aliaksei _S_everyn and Alessandro _M_oschitti. 2015. Learning to Rank Short Text Pairs with Convolutional Deep Neural 
+Networks. In Proceedings of the 38th International ACM SIGIR Conference on Research and Development in Information 
+Retrieval (SIGIR '15). ACM, New York, NY, USA, 373-382. DOI: http://dx.doi.org/10.1145/2766462.2767738
 
 
 ### Requirements
-gensim==1.0.1
+```
 nltk==3.2.2
 numpy==1.11.3
-pandas==0.19.2
-torch==0.1.11+b13b701
+pytorch==0.1.12
```

Comment:
These are outdated and can be removed, since there is already a `requirements.txt` file.